lib/Backends/CPU/libjit/libjit.cpp

```diff
@@ -1588,24 +1588,36 @@ libjit_dump_tensor(uint8_t *tensor, size_t *tensorDim, size_t numDimsTensor,
                    size_t elemKind, const char *name) {
   printf("%s\n", name);
   /// This definition should match the defintion in Glow.
```

Comment:
good catch! this is extremely easy to miss as it's likely we'd update ElemKind in Type.h first.
Can you add a comment there (type.h) that we need to take care about this definition in libjit as well.